Docs/Design/Plots.md

```
@@ -0,0 +1,10 @@
+https://github.com/davidfokkema/textual-plot
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)